crates/aptos-compression/src/metrics.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_metrics_core::{
+    register_histogram_vec, register_int_counter_vec, HistogramTimer, HistogramVec, IntCounterVec,
+};
+use once_cell::sync::Lazy;
+
+/// Useful metric constants for compression and decompression
+pub const COMPRESS: &str = "compress";
+pub const DECOMPRESS: &str = "decompress";
+pub const COMPRESSED_BYTES: &str = "compressed_bytes";
+pub const RAW_BYTES: &str = "raw_bytes";
+
+/// Counters for tracking the data compression ratio (i.e., total byte counts)
+pub static COMPRESSION_BYTE_COUNTS: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
If you ask me, I would remove the prefix "COMPRESSION_" from all these counter names.

Reply:
Done!